managed/CounterStrikeSharp.API/Modules/Menu/BaseMenu.cs

```diff
@@ -1,139 +1,118 @@
-﻿/*
- *  This file is part of CounterStrikeSharp.
- *  CounterStrikeSharp is free software: you can redistribute it and/or modify
- *  it under the terms of the GNU General Public License as published by
- *  the Free Software Foundation, either version 3 of the License, or
- *  (at your option) any later version.
- *
- *  CounterStrikeSharp is distributed in the hope that it will be useful,
- *  but WITHOUT ANY WARRANTY; without even the implied warranty of
- *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *  GNU General Public License for more details.
- *
- *  You should have received a copy of the GNU General Public License
- *  along with CounterStrikeSharp.  If not, see <https://www.gnu.org/licenses/>. *
- */
-
 using System.Collections.Generic;
+using CounterStrikeSharp.API.Modules.Utils;
+
+namespace CounterStrikeSharp.API.Modules.Menu;
 
-namespace CounterStrikeSharp.API.Modules.Menu
+public class BaseMenu : IMenu
 {
```

Comment:
Why have you changed the namespace? This is how it's done in the rest of the project so I'd rather keep it the same as everywhere else.